TINY-10015: implement size option for inline dialog (#8859)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

## Unreleased

### Fixed

> - Corrected the implementation of the size parameter in inline dialog. #TINY-10015

Reviewer: ```suggestion
- Corrected the implementation of the `size` parameter in inline dialogs. #TINY-10015
```

## 6.6.0 - TBA

### Fixed

> - Corrected the implementation of the `size` parameter in inline dialogs. #TINY-10015

Reviewer 2: We still don't technically support the 'large' size so wondering if we should be more specific
```suggestion
- Setting an inline dialog to a size of `'medium'` was not possible #TINY-10015
```

Author: Since it was not possible for both `normal` and `medium`, how about we address both?

```suggestion
- Fixed an issue that prevented users from setting the inline dialog `size` to 'medium' or 'normal'. #TINY-10015
```

Reviewer 2: Make sense 👍 One small tweak as we don't normally use the word 'users' or 'integrators' in vhangelog entries:
```
Fixed an issue that prevented setting the inline dialog `size` to 'medium' or 'normal'. #TINY-10015
```

Author: ```suggestion
- Fixed an issue that prevented setting the inline dialog `size` to 'medium' or 'normal'. #TINY-10015
```
